package.json

```diff
@@ -18,6 +18,7 @@
     "dev:sdk-demo": "turbo run dev --filter=@team-monite/sdk-demo",
     "build": "turbo run build",
     "clean": "rm -rf packages/*/dist",
+    "clean:node": "find . -name node_modules -type d -prune -exec rm -rf '{}' + && find . -name .turbo -type d -prune -exec rm -rf '{}' + && find . -name dist -type d -prune -exec rm -rf '{}' +",
```

Comment:
This doesn't seem related to this task, could you remove it?

Reply:
Yes, I can keep it in separate PR

Reply:
Better to keep them separated so we can properly track those later.